src/reducers/stories.reducer.js

```diff
@@ -0,0 +1,44 @@
+import initialState from './initialState'
+import {
+  STORIES_LOADED,
+  STORY_CREATED,
+  STORY_EDITED,
+  STORY_DELETED
+} from './../actions/actionTypes'
+
+export default function stories(state = initialState.stories, action) {
+  switch (action.type) {
+    case STORIES_LOADED:
+      return {
+        ...state,
+        stories: action.payload
+      }
+    case STORY_CREATED:
+      return {
+        ...state,
+        stories: [
+          state.stories,

```

Comment:
you want to copy all the existing stories in `state.stories` here, so you should do `...state.stories`